dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops

```diff
@@ -123,6 +123,15 @@ MESSAGE_FORMAT_UPDATED.useraction=No action is required.
 CONSOLE_FORMAT_UPDATED=TRAS3003I: The console output format has now been updated to '{0}'.
```

Comment:
Use "is" or "was" instead of "has been"

Reply:
hi Pam, I updated the both messages, please review the changes. Thanks